vowpalwabbit/core/src/reductions/audit_regressor.cc

```diff
@@ -28,10 +28,10 @@ static constexpr std::array<VW::column_definition, NUM_COLS> AUDIT_REGRESSOR_COL
 static const std::array<std::string, NUM_COLS> AUDIT_REGRESSOR_HEADER = {
     "example\ncounter", "values\naudited", "total\nprogress"};
 
-class audit_regressor_data
+class audit_regressor_data_obj
```

Comment:
A class shouldn't be named obj

Reply:
I updated this because I have been naming the data instance `blah_data`, I can update the class name to `audit_regressor` which is more standard